resources/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+cd "$THIS_SCRIPT_PATH"
+
+builder_describe "Keyman resources module" configure build test
```

Comment:
```suggestion
builder_describe "Keyman resources module" clean configure build test
```

We should include `clean` as one of our 'standard' actions (particularly as this is called as a 'child' script of /common/build.sh.